firmware/library/L4_Application/Temperature_Sensor.hpp

```diff
@@ -0,0 +1,17 @@
+// Interface used to obtain the temperature of the system.
```

Comment:
More like, sense the temperature of the temperature sensor.

Reply:
Got it, I'll update that.